templates/automotive/index.html

```diff
@@ -220,43 +65,348 @@ <h2 class="p-muted-heading u-align--center u-sv3">THESE AUTOMOTIVE INNOVATORS HA
+      <p>Scania deploys agile HPC clusters for rapid digital design, validation, crash testing, and aerodynamics.</p>
+      <p>Supercomputers on-demand for CAD, CAM and CAE. Tame SLURM and get more from your cluster.</p>
+      <p><a href="https://www.brighttalk.com/webcast/6793/416900/mastering-multicloud-with-slurm-and-juju" class="p-link--external">Watch the video&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/1c2a3c27-Automotive-Masters-vid-screen.png",
+          alt="",
+          height="400",
+          width="700",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row u-equal-height">
+    <div class="col-6 u-vertically-center u-align-center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/68d22036-Automotive-in-car-cockpit-cropped.svg",
+          alt="",
+          height="602",
+          width="766",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+    <div class="col-6">
+      <h2 class="p-heading--five">In-cabin digital</h2>
+      <p class="p-heading--two">Next level infotainment</p>
+      <p>Consumer expectations are relentless.</p>
+      <p>New insights. New apps. New content. New experiences. Personalised. Contextual.</p>
+      <p>The ever-fresh experience needs secure, reliable, over-the-air updates and <a href="https://snapcraft.io/store" class="p-link--external"></a>app stores.</p>
+      <p>NA new car feeling, every drive.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h2 class="p-heading--five">Over-the-air updates</h2>
+      <p class="p-heading--two">OTA? Compressed. Transactional. Signed. Simple.</p>
+      <p>Update any system, any device, any day. Future-proof your fleet.</p>
+      <p>Full OS containerisation means transactional, atomic updates. Data snapshot backup and restore. Signed software integrity. <a href="/engage/snap-deltas">Compressed delta updates</a>.</p>
+      <p><a href="/engage/snapd-whitepaper">Complete control</a>. No delayed patches, no partial updates, no unauthorised ECU software.</p>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/74859dc8-control-thin.svg",
+          alt="",
+          height="200",
+          width="205",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row u-equal-height">
+    <div class="col-6 u-vertically-center u-align-center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/0be66895-Automotive-car-sensor.svg",
```

Comment:
the image in the design has grey tyres?  also it is bigger in the design?

Reply:
Waiting @meralonso to provide the images. These are the ones in the copy doc.